server/index.js

```diff
@@ -24,10 +24,27 @@ app.post('/insertData', async (req, res) => {
 
 //wen
 app.post('/insertRunData', async (req, res) => {
-  const { vod, runtime, categoryID, gameName } = req.body;
+  const { vod, runtime, type, gameName } = req.body;
   try {
-    const result = await pool.query('INSERT INTO run (vod, runtime, categoryID, gameName) VALUES ($1, $2, $3, $4) RETURNING *', [vod, runtime, categoryID, gameName]);
-    console.log(`Data inserted with RunID: ${result.rows[0].runID}`);
+    const result = await pool.query('INSERT INTO run (vod, runtime, type, gameName) VALUES ($1, $2, $3, $4) RETURNING *', [vod, runtime, type, gameName]);
+    
+    if (result.rows.length > 0) {
```

Comment:
Could we possibly merge the two if statements and refactor our code so that the awaits happen one line after another?

The insertion of a run shouldn't be possible without it's submit -- therefore I think it's okay to bonk those tho lines together with an `&&`